dbms/src/Flash/tests/gtest_interpreter.out

```diff
@@ -0,0 +1,463 @@
+~func_name: SingleQueryBlock
+~result_index: 0
```

Comment:
Maybe we can log the test case content ahead of test result later.

Reply:
IMO, test_suite_name and result_index are enough.